lib/log/init.go

```diff
@@ -27,13 +29,18 @@ type Resource = otfields.Resource
 // For testing, you can use 'logtest.Init' to initialize the logging library.
 //
 // If Init is not called, Get will panic.
-func Init(r Resource) (sync func() error) {
+func Init(r Resource, sinks ...sinks.SinkCore) (sync func() error) {
```

Comment:
```suggestion
func Init(r Resource, sinks *sinks.Sinks) (sync func() error) {
```

Would _much_ prefer an implementation that does not expose `zapcore` to callers: https://github.com/sourcegraph/sourcegraph/pull/35879/files#r884010809